### ISSUE TITLE
⭐ [Documentation] - Fly.io has changed their pricing

### ISSUE THREAD

**Issue by Avi-Lance** [actualbudget/docs#415]
### Description

Fly.io has recently changed their free tier and pricing. At a minimum, assuming you have a shared-cpu-1x in Ashburn, Virginia (the cheapest region) and a 1 GB volume, it will be $2.09 a month. 

Source: https://fly.io/docs/about/pricing/

### Documentation URL

https://github.com/actualbudget/actual?tab=readme-ov-file#installation
https://actualbudget.org/docs/install/

**Comment by RubenOlsen** [comment]
I have consulted the fly.io pricing page, but could you please create a short spending set up to how the sum is 2.09 a month? This will aid me fixing the documentation.

**Comment by Avi-Lance** [comment]
shared-cpu-1x in Ashburn Virginia: $1.94 per Month
 Persistent Storage Volume: $0.15/GB per Month
 Total: $2.09 per Month
 
Compute Costs Screenshot: 
![image](https://github.com/user-attachments/assets/b36e73b8-fd49-4406-9684-db9409ebca17)


Storage Costs Screenshot:
![image](https://github.com/user-attachments/assets/8ef2909b-42ef-44a8-9658-a4a90477ef49)

**Comment by psybers** [comment]
In all honesty, I think the AB docs should avoid mentioning any concrete pricing.  Just mention they have a pay as you go plan and link to it.

Their prices will fluctuate too much.

**Comment by Jonathan-Fang** [comment]
I agree, but estimated pricing ranges would help ex. this hosting service is around X amount of money the last time we checked, please check the service for the latest update. Actual is not affiliated with fly.io.

If AB decides to update the pricing rates as external providers change, in my opinion so be it if people want to update it, but it helps to market AB to new users, especially non-tech-savvy users if there's a number that was true at one point in time; it would act like a hook.

**Comment by RubenOlsen** [comment]
I have considered this PR since it appeared "on the scene," I like @Jonathan-Fang's idea about a price range. This is manageable. I'll see what I can get done here.
